src/next-app/components/TextInput.tsx

```diff
@@ -15,6 +15,7 @@ import { BsChevronDown } from 'react-icons/bs';
 import DropdownField from './DropdownField';
 
 type TextInputProps = {
+  id: string;
```

Comment:
There are TypeScript errors present because of the addition of this id field. Can you update this to make `id` optional? That will address these issues. This should look like: `id?: string;`